cli/common/flag/set.go

```diff
@@ -56,6 +56,22 @@ func (f *Sets) NewSet(name string) *Set {
 	return flagSet
 }
 
+// GetSetFlags returns a slice of flags for a given set.
+// If the requested set does not exist, this will return an empty slice.
+func (f *Sets) GetSetFlags(setName string) []string {
```

Comment:
There was previously no way to get "all the flags in the 'foo' group of flags" where 'foo' could be "Global Options", "Command Options", etc. I added a method for that.